src/main/kotlin/com/ex_dock/ex_dock/database/account/AccountClasses.kt

```diff
@@ -32,7 +32,21 @@ enum class Permission(name: String) {
 
   companion object {
     fun fromString(value: String): Permission {
-      return values().find { it.name == value } ?: NONE
+      return when (value) {
+        "read" -> READ
+        "write" -> WRITE
+        "read-write" -> READ_WRITE
+        else -> NONE
+      }
+    }
+
+    fun toString(permission: Permission): String {
+      return when (permission) {
+        NONE -> "none"
+        READ -> "read"
+        WRITE -> "write"
+        READ_WRITE -> "read-write"
+      }
```

Comment:
This should be reverted.
The Permission.READ.toString() should be replaced with Permission.READ.name.
Also, the fromString function was implemented correctly in a way that would immediately support new values when added.